src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -105,7 +106,7 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
     * Executes task in given context.
     */
   override def execute(): Unit = {
-    notifyToSubscribers(executeTask)
+    notifyToSubscribers(executeTask())
```

Comment:
We should talk about conventions... I'm not saying this is wrong, just want to follow one code conventions guide.